src/symbols.js

```diff
@@ -16,20 +17,28 @@
  * accepted in (e.g. "math" or "text").
  */
 
-module.exports = {
-    math: {},
-    text: {},
+import type {Mode} from "./types";
+
+type Font = "main" | "ams";
+type Group =
+    "accent" | "bin" | "close" | "inner" | "mathord" | "op" | "open" | "punct" |
+    "rel" | "spacing" | "textord";
+type CharInfoMap = {[string]: {font: Font, group: Group, replace: ?string}};
+
+const symbols: {[Mode]: CharInfoMap} = {
+    "math": {},
+    "text": {},
 };
+export default symbols;
 
-function defineSymbol(mode, font, group, replace, name, acceptUnicodeChar) {
-    module.exports[mode][name] = {
-        font: font,
-        group: group,
-        replace: replace,
-    };
+/** `acceptUnicodeChar = true` is only applicable if `replace` is set. */
+function defineSymbol(
+    mode: Mode, font: Font, group: Group, replace: ?string, name: string,
+    acceptUnicodeChar?: boolean) {
```

Comment:
Ideally I'd like function args to formatted as so it's easier to read the args:
```
function foo(
   arg0: T0,
   arg1: T1,
   ...
   argn: Tn,
) {
   console.log('bar');
   ...
}
```
Unfortunately, I don't believe our babel setup supports trailing commas in function defintions and calls so let's leave this for now.  I opened an issue for this: https://github.com/Khan/KaTeX/issues/861.